uptane-standard.md

```diff
@@ -240,28 +277,222 @@ The Time Server SHALL expose a public interface allowing primaries to communicat
+    3. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+    4. Check that the version number in this targets metadata file matches the version number given for it in the snapshot metadata file. (Checks for a mix-and-match attack.)
+    5. Check that there are no delegations. (Targets metadata from the director MUST NOT contain delegations.)
+    6. Check that no ECU identifier is represented more than once.
+7. Download and check the root metadata file from the image repository:
+    1. Check that the metadata file has been signed by a threshold of keys specified in the previous root metadata file. (Checks for an arbitrary software attack.)
+    2. Check that the version number in the previous targets metadata file, if any, is less than or equal to the version number in this targets metadata file. (Checks for a rollback attack.)
+    3. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+    4. If the the timestamp and / or snapshot keys have been rotated, delete the previous timestamp and snapshot metadata files.
+8. Download and check the timestamp metadata file from the image repository:
+    1. Check that it has been signed by the threshold of keys specified in the root metadata file.
+    2. Check that the version number of the previous timestamp metadata file, if any, is less than or equal to the version number of this timestamp metadata file. (Checks for a rollback attack.)
+    3. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+9. Download and check the snapshot metadata file from the image repository:
+    1. Check that it has been signed by the threshold of keys specified in the root metadata file.
+    2. Check that the version number of the previous snapshot metadata file, if any, is less than or equal to the version number of this snapshot metadata file. (Checks for a rollback attack.)
+    3. Check that the version number the previous snapshot metadata file lists for each targets metadata file is less than or equal to the its version number in this snapshot metadata file.
+    4. Check that each targets metadata filename listed in the previous snapshot metadata file is also listed in this snapshot metadata file.
+    5. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+10. Download and check the top-level targets metadata file from the image repository:
+    1. Check that it has been signed by the threshold of keys specified in the root metadata file.
+    2. Check that the version number of the previous targets metadata file, if any, is less than or equal to the version number of this targets metadata file. (Checks for a rollback attack.) {#test}
```

Comment:
Is that "#test" intentional? Seems like a typo.

And do we really tolerate the version being equal? What's the point of versioning if we allow the version to remain constant?

Reply:
Answered by @JustinCappos below.